Add examples for BigInt API

Elchi3 opened the pull request:
Examples for:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/asIntN
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/asUintN
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString (only works in Chrome 76 for now, soon in Nightly)
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toString
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/valueOf

wbamberg commented on live-examples/js-examples/bigint/bigint-asintn.html:
```diff
@@ -0,0 +1,14 @@
+<code id="static-js">const max = 2n ** (64n - 1n) - 1n;
+
+function check64bit(number) {
+  (number > max) ? console.log("Number doesn't fit in signed 64-bit integer!") : console.log(BigInt.asIntN(64, number));
```
This line and the equivalent one for `asUintN()` is too long: as a rule of thumb I'd like them to be at most 1000px wide, so the user can see the whole thing without having to scroll. It's worse in this case too because the API being demonstrated isn't shown until character 99 of the line so it's very easy to miss.

I'd suggest formatting this line like:

```js
  (number > max) ?
    console.log("Number doesn't fit in signed 64-bit integer!") :
    console.log(BigInt.asIntN(64, number));
```

I think this also makes the branching logic of the line clearer.

Elchi3 replied:
This makes sense to me. I think I need to remember to have the local test interactive-examples not in a large window. Did we think about doing something about this? For example, setting a fixed width for testing purposes locally? Just a thought, though.

wbamberg replied:
We do have an issue for this: https://github.com/mdn/bob/issues/166 but I don't know when it might be fixed.

wbamberg commented on live-examples/js-examples/bigint/bigint-tolocalestring.html:
```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">let bigint = 123456789123456789n;
```
This could be `const`.

wbamberg commented on live-examples/js-examples/bigint/bigint-tostring.html:
```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">console.log(256n.toString());
+// expected output: "256"
```
It might be more illustrative to use the same bigint value for all three examples?

```js
console.log(1024n.toString());
// expected output: "1024"

console.log(1024n.toString(2));
// expected output: "10000000000"

console.log(1024n.toString(16));
// expected output: "400"
```

...just a suggestion though.

wbamberg commented on live-examples/js-examples/bigint/meta.json:
```diff
@@ -0,0 +1,34 @@
+        "bigIntasIntN": {
+            "exampleCode": "./live-examples/js-examples/bigint/bigint-asintn.html",
+            "fileName": "bigint-asintn.html",
+            "title": "JavaScript Demo: BigInt.asIntN",
```
This should be "JavaScript Demo: BigInt.asIntN()".

wbamberg commented on live-examples/js-examples/bigint/meta.json:
```diff
@@ -0,0 +1,34 @@
+        "bigIntasUintN": {
+            "exampleCode": "./live-examples/js-examples/bigint/bigint-asuintn.html",
+            "fileName": "bigint-asuintn.html",
+            "title": "JavaScript Demo: BigInt.asUintN",
```
This should be "JavaScript Demo: BigInt.asUintN()".

Elchi3 commented:
Thanks for the review, @wbamberg. Sorry to have poked about this one, but I thought it would be great to land it this week since we're shipping and advertising it. Addressed your comments. Great feedback as always!

Elchi3 commented:
Thanks for the quick fix! :heart:
